openmc/bounding_box.py

```diff
@@ -99,19 +106,64 @@ def extend(self, padding_distance: float) -> BoundingBox:
 
         Returns
         -------
-        An enlarged bounding box
+        An expanded bounding box
+        """
+        self[0] -= padding_distance
+        self[1] += padding_distance
+        return self
+
+    def expand(self, other_box: BoundingBox) -> BoundingBox:
```

Comment:
I would give this method an `inplace` argument as we do with other things. I think it is sensible to have the default be `inplace=False`.

Reply:
Oh hadn't seen that elsewhere (or forgot). I'll make it consistent w/ those other methods. Thanks!